qiskit_dynamics/models/__init__.py

```diff
@@ -12,40 +12,166 @@
+case of a Hamiltonian.
+
+The Schrodinger equation is represented via the
+:class:`~qiskit_dynamics.models.HamiltonianModel` class, which represents
+the decomposition:
+
+.. math::
+    H(t) = H_d + \sum_j s_j(t) H_j,
+
+where both :math:`H_j` and the *drift* :math:`H_d` are Hermitian operators, and
+:math:`s_j(t)` are either :class:`~qiskit_dynamics.signals.Signal` objects or
+are numerical constants. Constructing a :class:`~qiskit_dynamics.models.HamiltonianModel`
+requires specifying the above decomposition, e.g.:
+
+.. code-block:: python
+
+    hamiltonian = HamiltonianModel(operators, signals, drift)
+
+with ``operators`` being a specification of the :math:`H_j`, ``signals`` a specification of
+the :math:`s_j`, and ``drift`` a specification of :math:`H_d`.
+
+Similarly, the :class:`~qiskit_dynamics.models.LindbladModel` class represents
+the Lindblad equation as written above, with the Hamiltonian decomposed as in
+:class:`~qiskit_dynamics.models.HamiltonianModel`. It may be instantiated as
+
+.. code-block:: python
+
+    lindblad_model = LindbladModel(hamiltonian_operators,
+                                   hamiltonian_signals,
+                                   drift,
+                                   dissipator_operators,
+                                   dissipator_signals)
+
+where the arguments ``hamiltonian_operators``, ``hamiltonian_signals``, and ``drift`` are for
+the Hamiltonian decomposition as in :class:`~qiskit_dynamics.models.HamiltonianModel`,
+and the ``dissipator_operators`` correspond to the :math:`L_j`, and the ``dissipator_signals``
+the :math:`g_j(t)`, which default to the constant ``1.``.
+
+Once constructed, model classes enable *evaluation* of certain functions, e.g. for a
+:class:`~qiskit_dynamics.models.HamiltonianModel`, ``hamiltonian.evaluate(t)`` returns
+:math:`H(t)`, and ``hamiltonian.evaluate_rhs(t, y)`` evaluates :math:`-iH(t)y`. Similar
+behaviour applies to :class:`~qiskit_dynamics.models.LindbladModel`, however the
+:meth:`~qiskit_dynamics.models.LindbladModel.evaluate` method will raise an error unless
+a vectorized evaluation mode is set (see below).
+
+Rotating Frames
+^^^^^^^^^^^^^^^
+
+"Entering a rotating frame" is a common transformation on models of quantum systems.
+For example, for a Hamiltonian, this corresponds to the transformation
+
+.. math::
+    H(t) \mapsto e^{-tF}(H(t) - F)e^{tF}
 
-Quantum Models
-==============
+for an anti-Hermitian operator :math:`F`, called the *frame operator*,
+commonly expressed as a Hermitian operator via the association
+:math:`F = -iH`.
 
-Models for quantum systems.
+Any model class can be transformed into a rotating frame by setting the
+``rotating_frame`` property:
+
+.. code-block:: python
+
+    model.rotating_frame = frame_operator
+
+where ``frame_operator`` is specification of :math:`F=-iH`
+(either giving :math:`F` or :math:`H`) of valid type. Setting this property modifies
```

Comment:
Implemented changes:
- Described initial transformation in terms of Hermitian operator `H_0` (used `H_0` instead of `H` to avoid potential confusion with the original Hamiltonian `H(t)`).
- Stated the accepted types for `frame_operator` below the python code block, including `RotatingFrame`, and point the reader to "see below" for further description about `RotatingFrame`.
- Slightly expanded the blurb about `RotatingFrame` to state that internally it works with the anti-Hermitian matrix `F`, but that it can be specified in terms of `H_0` which will automatically be converted to `F`.